scripts/toaster_configuration

```diff
@@ -0,0 +1,153 @@
+#!/bin/sh
+#
+# Copyright 2015-2016 Mentor Graphics Corporation
+#
+# This file is licensed under the terms of the GNU General Public License
+# version 2.  This program  is licensed "as is" without any warranty of any
+# kind, whether express or implied.
+
+echo "Creating toaster configuration"
+
+BUILDDIR=$*
```

Comment:
This really makes no sense. If you pass multiple arguments, this will just error and freak out on you. Either you accept one argument, or you accept many, and in either case this needs to be quoted. `BUILDDIR="$1"` is appropriate.